build.gradle

```diff
@@ -16,6 +16,14 @@ dependencies {
 	implementation 'org.springframework.boot:spring-boot-starter'
 	testImplementation 'org.springframework.boot:spring-boot-starter-test'
 	testImplementation 'com.github.tomakehurst:wiremock-jre8:2.32.0'
+	implementation group: 'org.apache.httpcomponents', name: 'httpclient', version: '4.5.13'
+	testImplementation 'org.assertj:assertj-core:3.21.0'
+
+	implementation group: 'com.fasterxml.jackson.core', name: 'jackson-annotations', version: '2.13.0'
```

Comment:
Move version to 
ext {
	jacksonVersion = "2.13.0"
}

and if should be
implementation "com.fasterxml.jackson.core:jackson-annotations:${jacksonVersion}"
	implementation "com.fasterxml.jackson.core:jackson-core:${jacksonVersion}"
	implementation "com.fasterxml.jackson.core:jackson-databind:${jacksonVersion}"
	implementation "com.fasterxml.jackson.dataformat:jackson-dataformat-xml:${jacksonVersion}"
	implementation "com.fasterxml.jackson.core:jackson-databind:${jacksonVersion}"